src/main/resources/application.yml

```diff
@@ -38,6 +38,15 @@ spring:
         use_sql_comments: true
         format_sql: true
 
+  security:
+    oauth2:
+      client:
+        registration:
+          google:
+            clientId: 1057742566572-4ufig26uc1s8tiggp6ja3tf13s4iuo87.apps.googleusercontent.com
```

Comment:
what about the application-prod.yaml  ... these two configurations should be also present in the PROD version of the application.yaml

Reply:
I will add that.

Reply:
Updated